inc/alt-experience/init.php

```diff
@@ -29,3 +29,8 @@ function () {
 		);
 	}
 );
+
+$bust_mode_path = dirname( __DIR__, 2 ) . '/vendor/newfold-labs/wp-module-performance/includes/BurstSafetyMode/init.php';
```

Comment:
```suggestion
$bust_mode_path = BLUEHOST_PLUGIN_DIR . 'vendor/newfold-labs/wp-module-performance/includes/BurstSafetyMode/init.php';
```

This should be much more reliable.